indico/modules/events/registration/controllers/management/fields.py

```diff
@@ -0,0 +1,127 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+import json
+
+from flask import request, jsonify, flash, session
+
+from indico.core.db import db
+from indico.modules.events.registration import logger
+from indico.modules.events.registration.controllers.management.sections import RHManageRegFormSectionBase
+from indico.modules.events.registration.models.items import RegistrationFormText, RegistrationFormItem
+from indico.modules.events.registration.models.registration_form_fields import (RegistrationFormField,
+                                                                                RegistrationFormFieldData)
+from indico.util.i18n import _
+from indico.web.util import jsonify_data
+from MaKaC.webinterface.common.countries import CountryHolder
+
+
+class RHManageRegFormFieldBase(RHManageRegFormSectionBase):
+    """Base class for a specific field within a registration form"""
+
+    normalize_url_spec = {
+        'locators': {
+            lambda self: self.field
+        }
+    }
+
+    def _checkParams(self, params):
+        RHManageRegFormSectionBase._checkParams(self, params)
+        self.field = RegistrationFormItem.get_one(request.view_args['field_id'])
+
+
+class RHRegistrationFormToggleFieldState(RHManageRegFormFieldBase):
+    """Enable/Disable a field"""
+
+    def _process(self):
+        self.field.is_enabled = (request.args.get('enable') == 'true')
+        db.session.flush()
+        flash(_('Field "{title}" modified').format(title=self.field.title), 'success')
```

Comment:
Do we need those flash message? Isn't the UI quite clear (and do we even show those messages when they are returned in the json data to angular)